plugin/connect.go

```diff
@@ -83,12 +83,13 @@ func potentialPluginPaths(slug string) ([]string, error) {
 
 	// Assemble potential plugin paths
 	filename := fmt.Sprintf(
-		"nextmv-%s-%s-%s-%s-%s.so",
+		"nextmv-%s-%s-%s-%s-%s%s.so",
```

Comment:
Please see my comment on the private implementation. Not a blocker but maybe we can keep consistency of arguments being separated by a dash. Not a blocker though.
```suggestion
		"nextmv-%s-%s-%s-%s-%s-%s.so",
```

Reply:
It is separated by a dash, but the dash comes in on the variable.